tests/accidental_tests.ts

```diff
@@ -643,39 +714,43 @@ function sagittal(options: TestOptions): void {
   const notes = [
     f
       .StaveNote({ keys: ['d/4', 'f/4', 'b/4', 'b/4'], duration: '4' })
-      .addModifier(accid('accSagittal11MediumDiesisUp'), 1)
-      .addModifier(accid('accSagittal5CommaDown'), 2)
+      .addModifier(accid('\uE30A' /*accSagittal11MediumDiesisUp*/), 1)
+      .addModifier(accid('\uE303' /*accSagittal5CommaDown*/), 2)
```

Comment:
Just a note to self: Mohit and Aaron (and possibly others) are in favor of having friendlier ways to insert these unicode strings. So I can look into a good way to allow this.